microshift_install_get_ready/microshift-install-get-ready.adoc

```diff
@@ -16,3 +16,70 @@ include::modules/microshift-install-system-requirements.adoc[leveloffset=+1]
+//Q: should we have a RHEL link here as well?
+
+* Install the {oc-first} to access your cluster, see xref:../microshift_cli_ref/microshift-oc-cli-install.adoc#microshift-oc-cli-install[Installing the OpenShift CLI tool].
+
+* Understand the differences between the valid {op-system-base} installation types available, along with the support scope for each and the tools used.
+
+** Often, {op-system-ostree} is chosen for end-point devices with production clusters. This installation type can be done with network connections or offline, depending on the local environment.
+** Image mode for {op-system-base} is available as Technical Preview.
+** Sometimes, {op-system-rt-kernel} can be used where predictable latency is critical. Workload partitioning is also required for low-latency applications. See xref:../microshift_configuring/microshift_low_latency/microshift-low-latency.adoc#microshift-low-latency[Configuring low latency] for more information on low latency and the {op-system-rtk}.
+
+* When choosing an image-based installation, consider whether the destination hardware is intended to be in an offline or networked state, where you plan to build system images, and how you plan to load your {op-system-bundle} onto the hardware. Use the following scenarios as general guidance:
+
+** If you build either a fully self-contained {op-system-ostree} or image mode for {op-system-base} ISO outside a disconnected environment, then install the ISO locally on your edge devices, you likely do not need an RPM repository or a mirror registry.
+** If you build an ISO outside a disconnected environment that does not include the container images, but consists of only the RPMs, you need a mirror registry from which to pull container images.
+** If you build images inside a disconnected environment, or use package mode for installations, you need both a mirror registry and a simple file or directory-based RPM repository.
+
+Using RPMs, or package-based installation, of {microshift-short}::
+
+This is simple installation type uses a basic command to install {microshift-short} on an existing {op-system-base} machine. See xref:../microshift_install_rpm/microshift-install-rpm.adoc#microshift-install-rpm[Installing from an RPM package]. No other tools are required unless you are also installing a {op-system-base} system or virtual machine (VM) at the same time. For a package-based {op-system-base} installation, you can use {op-system-base} image builder.
+
+{op-system-base} image installation with {microshift-short} embedded::
+
+Image or container-based installation types involve creating an `rpm-ostree`-based version of {op-system-base} that is optimized for edge deployment. Familiarize yourself with the following {op-system-base} tools that are used for creating {op-system-ostree} images:
+
+* A Kickstart file, which provisions your specific operating system.
```

Comment:
using a kickstart file is not exclusive to "image mode", it could be usefull for all installation types.